src/markdown-pages/install-with-kurl/migrating-csi.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
This is a good example to have. Andrew Reed created a doc previously to talk about [migrating using snapshots](https://kurl.sh/docs/install-with-kurl/migrating). It would be good to make that a general purpose migration doc and talk about his cluster-to-cluster migration as well as this in-cluster migration example, and explain the two approaches.

If that ends up being too long of a page, you could split the two up and name them more specifically to juxtapose their purposes.

Reply:
I had initially written this as an addon to that existing migration page, and found it didn't flow well at all. I'll definitely add crosslinks